elated-egrets/bangazon-api
# Rm payment types

## DESCRIPTION
1. added payment types endpoint for api

## How to pull down
Commands

```
git fetch --all
git pull rm-payment-types
git checkout rm-payment-types
python manage.py runserver
```

## Steps to test
1. make sure payment types endpoint has been added and working.
1. endpoint should be able to list all payment types on /api/payments
1. endpoint should be able to have a detailed view on /api/payments/<id>

## What you changed
1. added files in models, views, and serializers directories for the payment endpoint,
1. added imports for these files in the __init__.py file in each directory
1. added url in /Api/urls.py for the viewset for payment

## What was added to read me
1. documentation added to wiki about payment type behavior

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
